src/coreclr/tests/scripts/run-pmi-diffs.py

```diff
@@ -180,7 +180,7 @@ def validate_args(args):
 
     coreclr_setup_args.verify(args,
                               "ci_arch",
-                              lambda ci_arch: ci_arch in coreclr_setup_args.valid_arches + ['x86_arm_altjit', 'x64_arm64_altjit'],
+                              lambda ci_arch: ci_arch in coreclr_setup_args.valid_arches + ['clrjit_win_arm_x86', 'clrjit_win_arm64_x64'],
```

Comment:
This file is currently unused, so I wouldn't touch it. However, if you do, I wouldn't touch this part: the `ci_arch` is a token, not a filename, passed in from the calling code.

Reply:
I made relevant changes down below to reflect this change. Is there anything else needed to make it correct?

Reply:
The file was written for the old Jenkins CI system, so the entire `ci_arch` concept doesn't really exist anymore... probably.

This file might possibly be able to be reused to some extent to add PR triggerable PMI-based asm diffs to the current AzDO CI, but certainly, many changes will be needed. So there's no point touching it right now.

Reply:
Got it. Reverted.